src/opengl/ogl_fbo.c

```diff
@@ -174,15 +174,18 @@ static void attach_depth_buffer(ALLEGRO_FBO_INFO *info)
       ALLEGRO_DISPLAY *display = _al_get_bitmap_display(info->owner);
       int w = al_get_bitmap_width(info->owner);
       int h = al_get_bitmap_height(info->owner);
+	  int samples = 0;
```

Comment:
Looks like there's some whitespace inconsistency.

Reply:
Fixed in the next commit. Thanks.